### DISCUSSION TITLE
Transfer ownership of an organization from a locked account

### DISCUSSION THREAD

**Discussion by konstantinmiller** [community/community#120820] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

We unfortunately lost access to the account that has the ownership of our GitHub organization. The access cannot be restored as we also lost access to the 2FA device. The organization has two further "ordinary" members. Is it possible to transfer the ownership to one of those members or any other account in such a situation? Or otherwise restore access to the owner account?

**Comment by ebndev** [comment]
Hi @konstantinmiller,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
